api/sds.proto

```diff
@@ -172,6 +172,13 @@ message CertificateValidationContext {
 
   // [#not-implemented-hide:] Must present signed certificate time-stamp.
   google.protobuf.BoolValue require_signed_certificate_timestamp = 6;
+
+  // An optional `certificate revocation list
+  // <http://https://en.wikipedia.org/wiki/Certificate_revocation_list>`_
+  // (in PEM format). If specified, Envoy will verify that the presented client
```

Comment:
This isn't restricted to client certificates, so please `s/client/peer/`.

Reply:
Done and done below!